.travis.yml

```diff
@@ -14,6 +14,7 @@ deploy:
     tags: true
 
 env:
+  - DEBUG="electron-installer-windows" NODE_VERSION="10"
```

Comment:
Also for `appveyor.yml`?

Reply:
Or perhaps we should try using Travis Windows support.

Reply:
> I'm just going to add support to AppVeyor

I added it: https://ci.appveyor.com/project/electron-userland/electron-installer-windows

I can add you to the AppVeyor org, I just need to figure out how to send an invitation.

> and in the future, I'll investigate Travis Windows and see if it's better than using AppVeyor.

I tried it with `electron-installer-zip` the other week (also in electron-userland) and it was hanging for some reason. The forums said it was because there were processes still running, so your mileage may vary.

Reply:
To send me an invite, go to `Settings` → `Team`